canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -123,12 +136,19 @@ class VirtualizedTable extends React.Component {
 
 	// Responsible for rendering the table header row given an array of columns.
 	headerRowRenderer(scrollKey, { className, columns, style }) {
+		const checkboxLabelColumnIndex = (typeof this.props.columns === "undefined" || this.props.columns.length === 0) ? 0 : this.getCheckboxLabelColumnIndex(this.props.columns);
```

Comment:
If there are no columns, will setting the index to 0 be an issue?

Reply:
This won't be a problem. If there are no columns, `headerCheckboxLabel` is set to an empty string.
```
const headerCheckboxLabel = (typeof this.props.columns === "undefined" || this.props.columns.length === 0) ? "" : this.props.columns[checkboxLabelColumnIndex].headerLabel;
```